packages/ui-library/src/components/input-field-text/index.ts

```diff
@@ -218,7 +219,7 @@ export class BlrInputFieldText extends LitElementCustom {
                 </div>
               `
             : nothing}
-          <div class="blr-input-wrapper ${inputContainerClasses}">
+          <div class="blr-input-wrapper ${inputContainerClasses}" ?readonly="${this.readonly}">
```

Comment:
in accordance with my first comment we should remove this attribute and add the disabled class to the `inputContainerClasses` object instead.

Reply:
I don't think we would need to remove this as disabled is already being applied to this class. The read only attribute is not being applied to this class hence i will keep it.